samples/nrf_desktop/src/modules/dfu.c

```diff
@@ -53,7 +60,7 @@ static void set_ble_latency(bool dfu_start)
 		const struct bt_le_conn_param param = {
```

Comment:
It should be noted that this problem would not have occured if one used 

#ifdef CONFIG_BT_PERIPHERAL

instead of

IS_ENABLED(CONFIG_BT_PERIPHERAL)
.

It's not obvious to me when to use one or the other, they each have their own problems. ifdef is safe from this class of bugs, but IS_ENABLED makes the code easier to read.

Reply:
True, I prefer to use `IS_ENABLED` unless it is really impossible. It gives the free compiler checks for whole code which is very useful when a lot of configuration macros is present in the system.